mcp/src/databricks_mcp/oauth_provider.py

```diff
@@ -0,0 +1,26 @@
+from databricks.sdk import WorkspaceClient
+from mcp.client.auth import OAuthClientProvider, TokenStorage
+from mcp.shared.auth import OAuthToken
+
+
+class DatabricksTokenStorage(TokenStorage):
+    def __init__(self, workspace_client):
+        self.workspace_client = workspace_client
+
+    async def get_tokens(self) -> OAuthToken | None:
+        headers = self.workspace_client.config.authenticate()
+        token = headers["Authorization"].split("Bearer ")[1]
+        return OAuthToken(access_token=token, expires_in=60)
```

Comment:
Is it guaranteed that these tokens expire in 60 min (is the unit here minutes)? Any way for us to tell? Otherwise picking a very short expiry time is probably best to ensure the token isn't inadvertently cached etc

Reply:
yep its in seconds, added a default variable name for more clarity